prometheus_hardware_exporter/collectors/lshw.py

```diff
@@ -0,0 +1,32 @@
+"""Lshw metrics collector."""
+import json
+from logging import getLogger
+from typing import Dict, List
+
+from ..utils import Command
+
+logger = getLogger(__name__)
+
+
+class LSHW(Command):
```

Comment:
should we cache results so we only subprocess out once? PSU ratings won't change over the lifetime of a server

Reply:
Add cache for dmidecode.